rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json

```diff
@@ -970,5 +970,24 @@
         }
       }
     ]
+  },
+  "step11": {
+    "title": "Sample collection",
+    "display_back_button": "true",
+    "next": "step1",
+    "bottom_navigation": "true",
+    "next_label": "OK",
+    "fields": [
+      {
+        "key": "lbl_high_heat_exposure",
+        "type": "label",
+        "text_color": "#000000",
+        "has_bg": true,
+        "bg_color": "#ffffff",
+        "center_label": "true",
+        "top_margin": "15dp",
+        "text": "This RDT has been flagged for high heat exposure and reliability has been compromised. Please use another RDT."
+      }
+    ]
```

Comment:
I think there's some misunderstanding here https://github.com/onaio/rdt-standard/pull/463#issuecomment-688092525.

There's no need to add an extra step, simply modify step2 to have this information `This RDT has been flagged for high heat exposure and reliability has been compromised. Please use another RDT.`

Reply:
fixed